docroot/core/phpunit.xml

```diff
@@ -27,9 +27,9 @@
     <!-- Example SIMPLETEST_DB value: mysql://username:password@localhost/databasename#table_prefix -->
     <env name="SIMPLETEST_DB" value="tcp://db:3306"/>
     <!-- Example BROWSERTEST_OUTPUT_DIRECTORY value: /path/to/webroot/sites/simpletest/browser_output -->
-    <env name="BROWSERTEST_OUTPUT_DIRECTORY" value=""/>
+    <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="/app/docroot/sites/simpletest/browser_output"/>
```

Comment:
Just a comment - I'd guess this is the kind of thing that circle can interpret and give us a nice summary of, so maybe one day we should look at saving this as an artefact.